### ISSUE TITLE
When grouping on an event itself the `group by` clause doesn' work

### ISSUE THREAD

**Issue by rgl** [espertechinc/nesper#5]
The following EPL does not work on NEsper above v4.8:

``` sql
select Service, count(*) as Value
from Service.win:time_batch(1 seconds)
group by Service
order by Service
```

Check the test at [EplTest.cs:GroupByRegression](https://github.com/rgl/UseNesper2/blob/master/csharp/UseNEsper2/EplTests.cs). If you checkout the [nesper-4.8 branch](https://github.com/rgl/UseNesper2/blob/nesper-4.8/csharp/UseNEsper2/EplTests.cs) (same code, only the nesper libraries are changed) it works fine.

If you comment the `group by Service` the statement generates output, so it seems `group by` is broken.

Can you reproduce the problem?

I'm using normal .NET 4.6 on Windows.

**Comment by rgl** [comment]
Just to add, it seems to be an issue with the handling of `null` values, as a workaround, doing `group by coalesce(Service, '')` seems to do the trick.

**Comment by rgl** [comment]
I've ported the [test to Java](https://github.com/rgl/UseNesper2/blob/master/java/src/test/java/com/example/EplTest.java). When using the latest version of Esper (5.3) it works fine. So this bug is specific to the csharp version.

**Comment by ajaxx** [comment]
Fixed in 5.5.0.
